hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/TargetManagementTest.java

```diff
@@ -721,4 +707,23 @@ public void targetCanBeReadWithOnlyReadTargetPermission() throws Exception {
         });
 
     }
+
+    @Test
```

Comment:
I would write: "Verify that the find all **targets** by ids **method** contains the entities **that we** are looking for"